### ISSUE TITLE
Additional limb-darkening laws

### ISSUE THREAD

**Issue by jdmonnier** [ExoCTK/exoctk#593]
Following up an email with Joe F:

Would it be possible to add two additional limb-darkening laws from Hestroffer 1997 

https://ui.adsabs.harvard.edu/abs/1997A%26A...327..199H/abstract

This simple 1-parameter model

I=I0 \mu^\alpha

works very well in some cases for interferometry data, though not sure how well it will work for transits.

I see Claret sometimes also uses the 2-parameter version (2-power) law also from Hestroffer

I/I0 = 1- g(1-mu^h). [when g=1 then reverts to the  previous 1-parameter version).

https://www.aanda.org/articles/aa/pdf/2023/06/aa46478-23.pdf

in conclusion, I am just writing to ask if it might be possible in future treatments to include these 2 additional limb-darkening models (power-law and 2-power-law from Hestroffer).  

Thanks for considering it,

-JDM
